vividus-plugin-mobile-app/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -139,7 +139,11 @@
         <constructor-arg index="0" value="${mobile.screenshot.downscale}" />
     </bean>
 
+    <bean class="org.vividus.mobileapp.converter.ParametersToMobileSequenceActionConverter" />
+    <bean class="org.vividus.mobileapp.converter.ExamplesTableToScreenshotConfigurationConverter" />
```

Comment:
the changes are mixed up across commits, just move it outside PR to avoid collisions